resources/homegrown/data-clinic-for-general.qmd

```diff
@@ -1,76 +1,268 @@
+
+
+# Words of wisdom & lessons learnt
+
+
+Since its inception, the Data Clinic has been a process of constant learning and development. We have shared below some of the lessons we have learnt, which may be relevant to your implementation. 
+
+1. Help with project methodology 
+
+The process of coming through Data Clinic often reveals that projects are not are thoroughly considered as they first appear (and this applies for all grades/seniority of Requestors!) or that the realities of the data available leads to significant reformulation of the project aims/methodology. 
+
+It is therefore been of enormous help to us to partner with Audit/QI/Methodology leads and experts within our department. We can now "refer" projects to these individuals if they need extra methodology support. Once they have been through this process, they come back to us with much clearer aims and reasoned data requests. 
+
+2. Types of requests accepted 
+
+We initially limited the scope of the Data Clinic to just serving requests for Audit or QI projects. 
+
+This served useful for limiting our workload while we developed our service. However, it quickly became apparent that many requests for projects that had clear potential benefits for the department did not fall into the category of Audit/QI. 
+
+As a result, we expended the scope of the projects we serve to:  
+- Audits  
+- QI  
+- Service evaluations  
+- Data for business cases  
+- Research feasibility projects  
+- Help with simple data analysis for existing datasets  
+- Others (at the discretion of whether the Data Clinic team feels it is reasonable)  
+
+This has increased the numbers of projects we serve and has certainly increased the usefulness of the Data Clinic to the department.
+
+However, it does have the downside of increasing workload, so perhaps caution is warranted when first starting. 
+
+
+# Contact details for the UCLH Data Clinic
+
+
+If you have questions on the above, or would like to discuss further, please email one of the individuals below (Dr Campbell will be the 1st point of call for most items). 
+
+Dr Ruaraidh Campbell: Lead Data Fellow for Data Clinic (ruaraidh.campbell@nhs.net)  
+Humayra Chowdhury: BI analyst (humayra.chowdhury@nhs.net)   
 Dr Tim Bonnici: Data Lead for Critical Care (t.bonnici@nhs.net)    
 
-Data Fellows: Change yearly, the fellows for 2022/23 are listed below  
 
-- Dr Ruaraidh Campbell (ruaraidh.campbell@nhs.net)
-- Dr Sarah Vaughan (s.vaughan5@nhs.net)
-- Dr Hrisheekesh Vaidya (h.vaidya@nhs.net)
+# Acknowledgements
+
+
+The Data Clinic has only been possible due to the contribution of numerous indivduals in addition to those listed above. We would like to acknowledge the time and effort that have put into this project. 
+
+Data Fellows (Past and Present): Dr Conor Foley [Lead Fellow for Data Clinic 2021/2022], Dr Dan Stein, Dr Hrisheekesh Vaidya, Dr Sarah Vaughan
+Business Intelligence Partners (Past and Present): Tracey Crissell, David Egan
```

Comment:
When looking at this on the example screen it doesn't put this as a separate line and merges it with the previous one